WearOAuth/util/src/main/java/com/example/android/wearable/oauth/util/Requests.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.android.wearable.oauth.util
```

Comment:
License header needed here!

Reply:
Ah, forgot to run Spotless on this module. Fixed!